File: matchmaker/server/src/main/java/ru/ifmo/se/dating/matchmaker/api/HttpAttitudesApi.java
package ru.ifmo.se.dating.matchmaker.api;

import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Service;

@Service
class HttpAttitudesApi implements AttitudesApiDelegate {

    @Override
    public ResponseEntity<Void> attitudesDelete(Long sourceId) {
        return ResponseEntityStub.create();
    }
}

File: matchmaker/server/src/main/kotlin/ru/ifmo/se/dating/matchmaker/api/HttpAttitudesApi.kt
package ru.ifmo.se.dating.matchmaker.api

import org.springframework.http.ResponseEntity
import org.springframework.stereotype.Service
import ru.ifmo.se.dating.matchmaker.api.generated.AttitudesApiDelegate

@Service
internal class HttpAttitudesApi : AttitudesApiDelegate {
    override fun attitudesDelete(sourceId: Long): ResponseEntity<Unit> =
        ResponseEntityStub.create()
}
